### PR TITLE
Fix incorrect Doxygen tag (@ince → @since). Doxygen parameter name matching.

### PR DESCRIPTION
**Pull request by practicalswift**
Doxygen fixes:
* Fix incorrect Doxygen tag (`@ince` → `@since`).
* Make Doxygen parameter names match actual parameter names.

### CODE REVIEW

**Comment by fanquake** [comment]
trivial ACK 8a2e5b1

**Comment by practicalswift** [comment]
@fanquake Thanks for the quick review. I added another commit to this PR (be9682d). Would you mind re-reviewing? :-)

File: src/wallet/rpcwallet.h [hunk 1]
```diff
@@ -16,7 +16,7 @@ void RegisterWalletRPCCommands(CRPCTable &t);
  * @param[in] request JSONRPCRequest that wishes to access a wallet
  * @return NULL if no wallet should be used, or a pointer to the CWallet
  */
-CWallet *GetWalletForJSONRPCRequest(const JSONRPCRequest&);
+CWallet *GetWalletForJSONRPCRequest(const JSONRPCRequest& request);
```
Comment (MeshCollider) [inline src/wallet/rpcwallet.h]: Why did you change this one and not the ones below it?

Comment (MeshCollider) [inline src/wallet/rpcwallet.h]: Oh don't worry you're just focusing on doxygen, sorry missed that

Comment (practicalswift) [inline src/wallet/rpcwallet.h]: They don't have any Doxygen documentation :-)

**Comment by MarcoFalke** [comment]
Please squash when this is ready for merge.

**Comment by practicalswift** [comment]
@MarcoFalke Squashed :-)

**Comment by MarcoFalke** [comment]
utACK 2c2e90d1d4768a7842b2fa840496cea567a8e995
